### PR TITLE
[ICATHALES-569][Dhyana] Update Dhyana rst file with Dhyana4040 detector model

### PR DESCRIPTION
**Pull request by sbahji**
[ICATHALES-569][Dhyana] Update Dhyana rst file with Dhyana4040 detector model

### CODE REVIEW

File: doc/Dhyana/dhyana.rst [hunk 1]
```diff
@@ -1,46 +1,79 @@
...
 
-This camera can manage 3 trigger modes : INTERNAL_SINGLE, EXTERNAL_MULTI, EXTERNAL_GATE
+Dhyana95
+-------------
 
-This camera can manage ROI 
+Introduction
+============
+| This is the reference documentation of the Dhyana Tango C++ device.
+| This camera can manage 4 trigger modes : INTERNAL_SINGLE, EXTERNAL_MULTI, EXTERNAL_GATE, INTERNAL_MULTI
```
Comment (soleil-ica) [inline doc/Dhyana/dhyana.rst:33]: triggers

File: doc/Dhyana/dhyana.rst [hunk 2]
```diff
@@ -1,46 +1,79 @@
...
++-----------------------------+-----------------------+
 
-This camera can manage 3 trigger modes : INTERNAL_SINGLE, EXTERNAL_MULTI, EXTERNAL_GATE
+Dhyana95
+-------------
 
-This camera can manage ROI 
+Introduction
+============
+| This is the reference documentation of the Dhyana Tango C++ device.
```
Comment (soleil-ica) [inline doc/Dhyana/dhyana.rst:32]: This is the documentation of the Lima/Dhyana Tango C++ device.

File: doc/Dhyana/dhyana.rst [hunk 3]
```diff
@@ -53,26 +86,33 @@ trigOutputDelay2                R/W                      Double             | Av
...
-Init                    Void            Void                    Do not use (Use Init of the Generic device)
+======================= =============== ======================= =============================================================================
+Init                    Void            Void                    Do not use (Use Init of the Generic device) 
 State                   Void            Long                    Returns the device state
-Status                  Void            String                  Returns the device state as a string
-setParamter             String Array    Void                    Set the value of a parameter
+Status                  Void            String                  Returns the device status
+setParamter             String Array    Void                    Sets the value of a parameter
 getParameter            String          String                  Returns the name and value of a specific parameter
 getAllParameter         Void            String                  Returns all parameters and their values
```
Comment (soleil-ica) [inline doc/Dhyana/dhyana.rst:117]: getAllParameters avec s je pense

File: doc/Dhyana/dhyana.rst (hunk 1 above)
Comment (sbahji) [reply doc/Dhyana/dhyana.rst]: Ok

File: doc/Dhyana/dhyana.rst (hunk 2 above)
Comment (sbahji) [reply doc/Dhyana/dhyana.rst]: Ok

File: doc/Dhyana/dhyana.rst (hunk 3 above)
Comment (sbahji) [reply doc/Dhyana/dhyana.rst]: exact

**Pull request merged**